docs/hypernode-platform/autoscaling/how-does-horizontal-autoscaling-work.md

```diff
@@ -160,9 +153,29 @@ After the version validation, please verify the MySQL host is set to `mysqlmaste
 
 You should see something similar to `'host' => 'mysqlmaster',`. If this is not the case please make sure the database connection host is set to `mysqlmaster` instead of `localhost` or `127.0.0.1` in the magento configuration file at `<magento_root>/app/etc/env.php`.
 
-### Make sure the Hypernode is a production plan
+### Application Specific Requirements - Magento2
 
-Unfortunately we don't support Horizontal autoscaling for development plans.
+#### Supported CMS
+
+Horizontal autoscaling is available for Magento2.
+To make use of Horizontal autoscaling, there are a couple of other requirements the application should meet.
+
+#### Make sure Elasticsearch/Opensearch configured properly
+
+Please make sure Elasticsearch or Opensearch host is set to `elasticsearchmaster` in the Magento2 configuration file at `<magento_root>/app/etc/env.php`
+More information about [Elasticsearch o Hypernode](../../hypernode-platform/tools/how-to-use-elasticsearch-on-hypernode.md)
+
+#### Make sure RabbitMQ configured properly
+
+Please make sure RabbitMQ host is set to `rabbitmqmaster` in the Magento2 configuration file at `<magento_root>/app/etc/env.php`
+More information about [RabbitMQ o Hypernode](../../best-practices/database/how-to-run-rabbitmq-on-hypernode.md)
+Also make sure to configured RabbitMQ without guest user.
+
```

Comment:
Either here or in the rabbitmq documentation page it should be clear that there's a non-default rabbitmq provisioned by default, called `hypernode-admin`, but users are also free to create new users if they're more comfortable with that.

Reply:
Added the info to both the docs. The more the merrier 🎅